crates/subspace-networking/src/create.rs

```diff
@@ -34,6 +34,28 @@ use tracing::info;
 const KADEMLIA_PROTOCOL: &[u8] = b"/subspace/kad/0.1.0";
 const GOSSIPSUB_PROTOCOL_PREFIX: &str = "subspace/gossipsub";
 
+/// Defines relay configuration for the Node
+#[derive(Clone)]
+pub enum RelayMode{
+    /// No relay configured.
+    NoRelay,
+    /// The node enables the relay behaviour.
+    Server,
+    /// Client relay configuration (enables relay client behavior).
+    /// It uses a circuit relay server address as a parameter.
+    ///
+    /// Example: /memory/<port>/p2p/<server_peer_id>/p2p-circuit
+    Client(Multiaddr)
+}
+
+impl RelayMode {
+    /// Defines whether the node has its relay behavior enabled.
+    pub fn is_relay_server_enabled(&self)-> bool{
+        matches!(self, RelayMode::Server)
+    }
+}
+
+
```

Comment:
Just one line. Did `rustfmt` integration broke for you?

Reply:
Fixed.